src/make_csv.py

```
@@ -237,40 +239,55 @@ def get_orcasite_detections(feed: OrcasiteFeed) -> List[OrcasiteDetection]:
+            # If no data is returned, we've fetched all pages
+            if not items:
+                break
+
+            for item in items:
+                attrs = item.get("attributes", {})
+
+                # Parse timestamp safely
+                ts_raw = attrs.get("timestamp")
+                try:
+                    ts = datetime.fromisoformat(ts_raw.replace("Z", "+00:00"))
+                except Exception:
```

Ruff (0.14.10)

Do not catch blind exception: `Exception`

(BLE001)